qiskit_nature/algorithms/ground_state_solvers/minimum_eigensolver_factories/vqe_ucc_factory.py

```diff
@@ -209,29 +277,21 @@ def get_solver(  # type: ignore[override]
         ansatz.num_particles = num_particles
         ansatz.num_spin_orbitals = num_spin_orbitals
         ansatz.initial_state = initial_state
+        self._vqe.ansatz = ansatz
 
         if isinstance(self.initial_point, InitialPoint):
             self.initial_point.grouped_property = driver_result
             self.initial_point.ansatz = ansatz
-
             # Override the initial_point with the computed array.
             self.initial_point = self.initial_point.to_numpy_array()
```

Comment:
I would not overwrite `self.initial_point` here and instead simply have a local `init_point` variable (or similarly named).

Reply:
Do you mean you would make a copy of self.initial_point, modify it and then pass it to the vqe?

Reply:
Simply write `initial_point = self.initial_point.to_numpy_array()` and then use this variable below, instead. This would obviously need to handle the `else` clause to this if-statement, too.

Reply:
Got it